containers/Ideas/IdeaProject.js

```diff
@@ -84,11 +105,13 @@ export const IdeaProject = (props) => {
                 <View style={styles.listContainer}>
                   <IdeasList ideas={ideas} {...props} />
                 </View>
-                <Button
-                  buttonStyle={styles.submitButton}
-                  title='Submit Idea'
-                  onPress={pressHandler}
-                />
+                {activePhase && (
```

Comment:
It should only appear in idea collect, not in rating phase. I meant this field for it: https://github.com/liqd/adhocracy-plus/blob/240273e6d6f503786df40a4dab7c47bc2fbf14aa/apps/projects/serializers.py#L99
But that's for after review then!